src/pybind/mgr/dashboard/controllers/user.py

```diff
@@ -48,6 +48,11 @@ def create(self, username=None, password=None, name=None, email=None, roles=None
             raise DashboardException(msg='Username is required',
                                      code='username_required',
                                      component='user')
+        if not password:
+            raise DashboardException(msg='Password is required',
```

Comment:
We cannot make the password field required because we may be using Single-Sign-On

Reply:
@ricardoasmarques  ok, but what if someone will create user without password? Then it is not possible to login on that user.

Reply:
@dziomdziorae If the user doesn't have a password, he will not be able to authenticate "locally" [1], he will only be able to authenticate "externally" (SSO) [2].

[1] "locally" - using ceph dashboard internal user database
[2] "externally" - using an external identity provider when Single-Sign-On is enabled - http://docs.ceph.com/docs/master/mgr/dashboard/#enabling-single-sign-on-sso

Reply:
Ok, thank you for explanation